src/browser/modules/GuideCarousel/GuideCarousel.tsx

```diff
@@ -65,15 +58,6 @@ function GuidesCarousel({
     }
   }
 
-  function onKeyUp(e: React.KeyboardEvent) {
```

Comment:
Was the reasoning for this that using left/right arrow makes sense in a frame, but not in a sidebar? There's a tiny chance people will notice the inconsistency that you can use keyboard shortcuts in the main play guides, but not the sidebar guides.

Reply:
I agree, and after not getting the keyEvents to fire in this component I gave up and removed it to cut scope for the release. Should have documented that properly! Hope to get the arrow keys working again before the release but I had more pressing matters 😓

Reply:
Feel free to give it a try, I might have missed something obvious